cmd/openfga/main_test.go

```diff
@@ -494,6 +522,76 @@ func connect(t *testing.T, tester OpenFGATester) *grpc.ClientConn {
 	return conn
 }
 
+// MigrateDatastoreToVersion5Test is meant to prove that a server that hasn't been updated to the latest version
+// can still write tuples after the database has been migrated to version 5
+func MigrateDatastoreToVersion5Test(t *testing.T, engine string) {
+	t.Run(engine, func(t *testing.T) {
+		store := ulid.Make().String()
+		t.Logf("start the database container and run migrations up to the latest migration version available")
+		testDatastore := storagefixtures.RunDatastoreTestContainer(t, engine)
+		uri := testDatastore.GetConnectionURI(true)
+
+		t.Logf("migrate database down to version 4")
+		migrateCommand := migrate.NewMigrateCommand()
+		migrateCommand.SetArgs([]string{"--datastore-engine", engine, "--datastore-uri", uri, "--version", strconv.Itoa(4)})
+		err := migrateCommand.Execute()
+		require.NoError(t, err)
+
+		t.Logf("start openfga 1.3.3 and wait for it to be ready")
+		// this is a hack to have the networking work in GitHub Actions
+		datastoreConnectionURI := strings.Replace(uri, "localhost", defaultIPAddressInLinux, -1)
```

Comment:
@miparnisari why is this an issue on this test but not on other tests relying on datastore tests containers?

Reply:
The test failed exactly here... need to investigate

Reply:
i wonder if it's something to do with how we setup the host config for that openfgatester https://github.com/openfga/openfga/blob/main/cmd/openfga/main_test.go#L91-L95